assets/presto_parquet/ingest_pairwise_arrays.py

```
@@ -0,0 +1,109 @@
+    result = {
+        "string": {"columns": [], "values": []},
+        "int": {"columns": [], "values": []},
+        "float": {"columns": [], "values": []},
+        "bool": {"columns": [], "values": []}
+    }
+
+    def recurse(sub_obj, current_path):
+        if isinstance(sub_obj, dict):
+            for k, v in sub_obj.items():
+                new_path = f"{current_path}.{k}" if current_path else k
+                assert sub_obj != v
```

Ruff (0.12.2)

Use of `assert` detected

(S101)